go/client/cmd_pgp_list.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol"
+)
+
+func NewCmdPGPList(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "list",
+		Usage: "List the active PGP keys in your account.",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdPGPList{Contextified: libkb.NewContextified(g)}, "list", c)
+		},
+	}
+}
+
+type CmdPGPList struct {
+	libkb.Contextified
+}
+
+func (s *CmdPGPList) ParseArgv(ctx *cli.Context) error {
+	if len(ctx.Args()) > 0 {
+		return UnexpectedArgsError("pgp list")
+	}
+
+	return nil
+}
+
+func (s *CmdPGPList) Run() error {
+	configCli, err := GetConfigClient(s.G())
+	if err != nil {
+		return err
+	}
+
+	currentStatus, err := configCli.GetCurrentStatus(context.TODO(), 0)
+	if err != nil {
+		return err
+	}
+	if !currentStatus.LoggedIn {
+		return libkb.LoginRequiredError{}
+	}
+
+	userCli, err := GetUserClient()
+	if err != nil {
+		return err
+	}
+
+	publicKeys, err := userCli.LoadPublicKeys(context.TODO(), keybase1.LoadPublicKeysArg{Uid: currentStatus.User.Uid})
+	if err != nil {
+		return err
+	}
+
+	dui := s.G().UI.GetDumbOutputUI()
+	for _, key := range publicKeys {
+		if len(key.PGPFingerprint) == 0 {
+			continue
+		}
+		dui.Printf("Keybase Key ID:  %s\n", key.KID)
```

Comment:
One more bikeshedding thought: we've unfortunately used "key_id" to refer to a shortened fingerprint in a few places, like [in the sigchain](https://keybase.io/oconnor663/chain). That be a reason to prefer "KID" here, even though it's less clear by itself? Not sure.